tools/build/wheel_linux_before_all.sh

```diff
@@ -1,25 +1,27 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
Normally Bash isn't installed out-of-the-box on Alpine images. But it seems like the `quay.io/pypa/musllinux_*` has it. I could revert back to `/bin/bash` if you want? Otherwise this script is POSIX now, which is probably a good thing since it needs to run on multiple platforms.

Reply:
It seems like you have dropped the checks for undeclared variables and failing pipes. The script contains variables and pipes. If you don't mind I would prefer it to run in safe mode :)

Reply:
I can bring back `uo` (shouldn't have removed those!). But I can't add back `pipefail` without reverting back to Bash. Which is why I changed the code to not use any pipes. But it's your call!

Reply:
I prefer to have pipefail over useing sh.

Reply:
Fair enough!